sources/rest_api/__init__.py

```diff
@@ -347,7 +361,7 @@ def paginate_dependent_resource(
                         if parent_record:
                             for child_record in child_page:
                                 child_record.update(parent_record)
-                        yield child_page
+                        yield from child_page
```

Comment:
```suggestion
                        yield child_page
```
In this case we actually need to `yield` the page data